python_modules/libraries/dagster-slack/dagster_slack/resources.py

```diff
@@ -0,0 +1,37 @@
+from slackclient import SlackClient
+
+from dagster import seven
+from dagster import Dict, Field, ResourceDefinition, String
+
+
+class SlackConnection:
+    def __init__(self, token):
+        self.token = token
+        self.sc = SlackClient(self.token)
+
+    def post(
```

Comment:
i think our naming convention for methods should directly mirror the namespace of their method calls. 

connection.chat.post_message

may seem natural

And I think a 

connection.api_call()

that does a direct pass through to the slack client make sense.

Reply:
yep, agree - updated